test/Microsoft.OpenApi.Readers.Tests/V3Tests/OpenApiDiscriminatorTests.cs

```diff
@@ -38,6 +38,7 @@ public async Task ParseBasicDiscriminatorShouldSucceed()
                    Mapping =
                     {
                             ["puppy"] = new OpenApiSchemaReference("Dog", openApiDocument),
+                            ["kitten"] = new OpenApiSchemaReference("Cat" , openApiDocument, "Cat"),
```

Comment:
```suggestion
                            ["kitten"] = new OpenApiSchemaReference("Cat" , openApiDocument, "https://gigantic-server.com/schemas/animals.json"),
```

Sorry, this is what I meant.